docs/HOWTO-use-texter-sideboxes.md

```diff
@@ -75,6 +75,10 @@ Mobilize (America) event scheduling. User clicks the button and the MOBILIZE_EVE
 
 Note: as of 3/1/21 you will need to reach out to Mobilize support to have them enable embedding for your dashboard(s), otherwise you'll probably get the error `Refused to display '{MOBILIZE_EVENT_SHIFTER_URL}' in a frame because it set 'X-Frame-Options' to 'sameorigin'.`
 
+### per-campaign-bulk-send
```

Comment:
This is a texter sidebox? Does this mean that an individual can turn this on and off per campaign?

Reply:
Yes